libkbfs/tlf_journal.go

```diff
@@ -822,6 +822,10 @@ func (j *tlfJournal) removeFlushedMDEntry(ctx context.Context,
 		return err
 	}
 
+	if err := j.blockJournal.onMDFlush(ctx); err != nil {
```

Comment:
Worth a brief comment, I think. Something like:

// This isn't with the call to j.onMDFlush since it needs to be under lock.